### PR TITLE
task done

### PR DESCRIPTION
**Pull request by maksym-mishchanchuk**
In my opinion, the task is done correctly, but there is a problem with the test. Please look

- [DEMO LINK](https://maksym-mishchanchuk.github.io/layout_stars/)
- [TEST REPORT LINK](https://maksym-mishchanchuk.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,49 @@
...
   <body>
-    <h1>Stars</h1>
+    <div class="container-stars">
+      <div class="stars--0 stars__star">
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+        <div></div>
+      </div>
```
Comment (Anna-Tkachenko) [inline src/index.html:18]:
```suggestion
      <div class="starts stars--0">
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
      </div>
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  background-position: center;
+}
+
+.stars--2 > :nth-last-child(-n+3) {
+  width: 16px;
+  height: 16px;
+  background-image: url(/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
```
Comment (Anna-Tkachenko) [inline src/style.css:65]: Check point 2 in [checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md)

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
+  display: flex;
+  justify-content: space-between;
+}
```
Comment (Anna-Tkachenko) [inline src/style.css:20]:
```suggestion
.stars {
  display: flex;
}
```

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+.stars--0 > .stars__star {
+  width: 16px;
```
Comment (Anna-Tkachenko) [inline src/style.css:23]: set margin-right for each star except the last

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--5 > .stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(/images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```
Comment (Anna-Tkachenko) [inline src/style.css:99]: Check 2 point in checklist

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+  background-image: url(/images/star-active.svg);
+}
+
+.stars--4 > :not(:last-child) {
+  background-image: url(/images/star-active.svg);
+}
+
+.stars--5 > .stars__star {
+  background-image: url(/images/star-active.svg);
+}
```
Comment (OleksiiBlezniuk) [inline src/style.css:37]: Не исправил, посмотри 2-ой пункт в чеклисте

**Review by OleksiiBlezniuk — CHANGES_REQUESTED** [review]

**Review by OleksiiBlezniuk — APPROVED** [review]
Good job! 🚀

**Pull request closed without merging**